specs/features/landing_feature.js

```diff
@@ -0,0 +1,32 @@
+var Setup = require('../features/helpers/setup');
+
+describe('Landing page', function() {
```

Comment:
you can "register" babel and use it for tasks/tests/whatever, see: https://babeljs.io/docs/usage/require/